crates/uv/src/commands/pip_install.rs

```diff
@@ -679,6 +682,10 @@ async fn install(
         }
     }
 
+    if compile {
+        compile_and_report(&mut printer, venv).await?;
+    }
```

Comment:
Let's move this above the change reporting, maybe? So that this summary comes at the end?